### ISSUE TITLE
Build failure with jakarta.jdt project

### ISSUE THREAD

**Issue by scottkurz** [eclipse/lsp4jakarta#439]
I tried to bump the JDT LS version after noticing that the underlying target platform was pointing to an interim/early Eclipse 4.27 build.   Now that 4.27 has been released I guess they removed those repo links.

Fixed by: https://github.com/eclipse/lsp4jakarta/pull/438

But hit this with `mvn install` of the `jakarta.jdt` subproject:

```text
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:2.7.2:validate-classpath (default-validate-classpath) on project org.eclipse.lsp4jakarta.jdt.test: Execution default-validate-classpath of goal org.eclipse.tycho:tycho-compiler-plugin:2.7.2:validate-classpath failed: org.osgi.framework.BundleException: Bundle org.eclipse.lsp4jakarta.jdt.test cannot be resolved:org.eclipse.lsp4jakarta.jdt.test [158]
[ERROR]   Unresolved requirement: Require-Bundle: org.eclipse.lsp4jakarta.jdt.core
[ERROR]     -> Bundle-SymbolicName: org.eclipse.lsp4jakarta.jdt.core; bundle-version="0.1.1.qualifier"; singleton:="true"
[ERROR]        org.eclipse.lsp4jakarta.jdt.core [157]
[ERROR]          Unresolved requirement: Require-Bundle: org.eclipse.m2e.core
[ERROR]            -> Bundle-SymbolicName: org.eclipse.m2e.core; bundle-version="2.0.7.20230307-1553"; singleton:="true"
[ERROR]               org.eclipse.m2e.core [135]
[ERROR]                 Unresolved requirement: Require-Bundle: org.eclipse.m2e.maven.runtime; bundle-version="[3.8.6,4.0.0)"
[ERROR]                   -> Bundle-SymbolicName: org.eclipse.m2e.maven.runtime; bundle-version="3.8.701.20230209-1606"
[ERROR]                      org.eclipse.m2e.maven.runtime [137]
[ERROR]                        Unresolved requirement: Import-Package: javax.annotation; version="[1.2.0,2.0.0)"
[ERROR]
```

**Comment by evie-lau** [comment]
Able to build when bumping up `<tycho.version>` to 3.0.3

**Comment by scottkurz** [comment]
Fixed by: #438
